src/components/auth/SocialAuthButton.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+const { REACT_APP_GOOGLE_LOGIN, REACT_APP_FACEBOOK_LOGIN, REACT_APP_TWITTER_LOGIN } = process.env;
```

Comment:
Good forward thinking abstracting this out 💪

Reply:
Thanks.